internal/pkg/plugin/gitlabci/golang/gitlabci.go

```diff
@@ -0,0 +1,55 @@
+package golang
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/merico-dev/stream/internal/pkg/log"
+)
+
+const ciFileName string = ".gitlab-ci.yml"
```

Comment:
const (
    ciFileName string = ".gitlab-ci.yml"
    commitMessage string = "managed by DevStream"
)

maybe it's better